File: sandbox.client/src/model/Charm.ts
import {ClientRecord} from "./ClientRecord";

export class Charm{

  public id : number;
  public name : string;
  public description: string;
  public energy:number;
  public actual;

  public static create(a : any): Charm {
    const ret = new Charm();
    ret.assign(a);
    return ret;
  }

  assign(a : any){
    console.log(a);
    this.id = a.id;
    this.name = a.name;
    this.description = a.description;
    this.energy = a.energy;
    this.actual = a.actual;
  }

}

File: sandbox.controller/src/kz/greetgo/sandbox/controller/model/Charm.java
package kz.greetgo.sandbox.controller.model;

public class Charm {
    public Long id;
    public String name;
    public String description;
    public Double energy;
    public Boolean actual;


    public Charm(Long id, String name, String description, Double energy,Boolean actual) {
        this.id = id;
        this.name = name;
        this.description = description;
        this.energy = energy;
        this.actual = actual;
    }

    public Charm(){}

    @Override
    public String toString() {
        return "Charm{" +
                "id=" + id +
                ", name='" + name + '\'' +
                ", description='" + description + '\'' +
                ", energy=" + energy +
                ", actual=" + actual +
                '}';
    }
}
